package.json

```diff
@@ -17,7 +17,17 @@
     "consume": "ts-node src/consume",
     "deploy": "ts-node src/deploy-contract",
     "send-tx": "ts-node src/send-tx",
-    "orchestrate": "orchestrate"
+    "orchestrate": "orchestrate",
+    "deps": "docker-compose -f scripts/deps/docker-compose.yml up -d",
+    "down-deps": "docker-compose -f scripts/deps/docker-compose.yml down --volumes --timeout 0",
+    "up-orchestrate": "docker-compose up -d",
+    "stop-orchestrate": "docker-compose stop",
+    "down-orchestrate": "docker-compose down --volumes --timeout 0",
+    "up": "npm run deps && npm run orchestrate-up",
```

Comment:
I guess it should be:
```
"up": "npm run deps && npm run up-orchestrate",
```

Reply:
Yes thanks. I changed it after testing it... sorry